src/vcpkg/base/system.cpp

```diff
@@ -372,4 +372,22 @@ namespace vcpkg
 namespace vcpkg::Debug
 {
     std::atomic<bool> g_debugging(false);
+
+    void println(StringView sv)
+    {
+        if (g_debugging)
+        {
+            msg::write_text_to_stdout(Color::None, "[DEBUG] ");
+            msg::write_text_to_stdout(Color::None, sv);
+            msg::write_text_to_stdout(Color::None, "\n");
```

Comment:
```suggestion
            msg::write_text_to_stdout(Color::None, Strings::concat("[DEBUG] ", sv, "\n");
```

Message printing performance on Windows is _atrocious_ and in `--debug` mode combining these print calls will run significantly faster.